src/sections/Resources/Resources-grid/filters.js

```diff
@@ -69,138 +72,155 @@ const Navigation = (props) => {
   const meshesFieldValues = meshes.map(mesh => mesh.fieldValue);
 
 
-
   return (
     <ResourceNavigationWrapper>
       <div className="filter">
+        <div onClick={() => {
+          setExpandFilter(!expandFilter); 
+        }} className="filter-menu-icon">
+          {expandFilter ? (<>
+            <IoClose className="filter-icon" />
+            <span>Close Filters</span>
```

Comment:
![clearafter](https://user-images.githubusercontent.com/85789734/149321817-e6ade112-d9c8-4f82-b435-19b83f16397f.jpg)

We still need the `Clear Filters` option like we have now. 

![clearbefore](https://user-images.githubusercontent.com/85789734/149321798-8a9da854-97e2-49eb-b15d-03c1a68c84df.jpg)